src/platform/packages/shared/kbn-alerts-as-data-utils/src/field_maps/alert_field_map.ts

```diff
@@ -320,6 +321,11 @@ export const alertFieldMap = {
     array: false,
     required: false,
   },
+  [ALERT_MUTED]: {
```

Comment:
One of the acceptance criteria is

> The underlying API for fetching alerts supports filtering by the kibana.alert.muted field.

I am assuming declaring this field here makes it searchable. Is that right?